tests/integration_tests/common.py

```diff
@@ -17,3 +20,32 @@ def wait_port(port_num, timeout):
         in_use = is_port_in_use(port_num)
 
     return in_use
+
+def prepare_config(config):
+    for key in config._config['integrations'].keys():
+        config._config['integrations'][key]['enabled'] = key == 'default_mariadb'
+
+    TEMP_DIR = pathlib.Path(__file__).parent.absolute().joinpath('../temp/').resolve()
+
+    config.merge({
+        'interface': {
+            'datastore': {
+                'storage_dir': str(TEMP_DIR.joinpath('datastore/'))
+            },
+            'mindsdb_native': {
+                'storage_dir': str(TEMP_DIR.joinpath('predictors/'))
+            }
+        }
+    })
+
+    if not os.path.isdir(config['interface']['datastore']['storage_dir']):
+        os.makedirs(config['interface']['datastore']['storage_dir'])
+    
+    if not os.path.isdir(config['interface']['mindsdb_native']['storage_dir']):
+        os.makedirs(config['interface']['mindsdb_native']['storage_dir'])
```

Comment:
Mindsdb should create these directories on boot, if it doesn't, that's an issue and should be fixed, if it does, these lines can be removed.

Reply:
issue added
https://github.com/mindsdb/mindsdb/issues/557